modules/apps/depot/depot-test/src/testFunctional/tests/MyDMTest.testcase

```diff
@@ -0,0 +1,78 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Henrique Page");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page");
+
+			User.logoutPG();
+		}
+	}
+
+	@description = "Can Edit a Documents and Media"
+	@priority = 5
+	test CanDeleteDM {
+		task ("AND create 2 Documents and Media (Upload the file)") {
+			DMDocument.addCP(
+				dmDocumentFile = "Document_3.txt",
+				dmDocumentTitle = "DM1 Test");
+
+			DMDocument.addCP(
+				dmDocumentFile = "Document_4.txt",
+				dmDocumentTitle = "DM2 Test");
+		}
+
+		Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
```

Comment:
We don't have any steps established outside of the Given-When-Then pattern, add this step inside the Then task block, since it's part of the Then process